master/static/srv/notebook-template.ipynb

```diff
@@ -24,6 +24,25 @@
         "### Environment"
       ]
     },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {},
+      "outputs": [],
+      "source": [
+        "# JupyterLab Notebooks in Determined",
+        "\n",
+        "     # Determined makes managing the environment and resources of JupyterLab easy by using containerized instances.\n",
+        "     # For more information on Determined, please look at our documentation (https://docs.determined.ai/latest/).\n",
```

Comment:
This is a bit tricky because this might show up in a dev version. So either this should use a relative path or you should delete this line.